src/test/java/com/google/api/gax/grpc/SettingsTest.java

```diff
@@ -126,16 +124,15 @@
     }
 
     private final SimpleCallSettings<Integer, Integer> fakeMethodSimple;
-    private final PageStreamingCallSettings<Integer, Integer, FakePagedListResponse>
+    private final PagedCallSettings<Integer, Integer, FakePagedListResponse>
         fakeMethodPageStreaming;
     private final BundlingCallSettings<Integer, Integer> fakeMethodBundling;
 
     public SimpleCallSettings<Integer, Integer> fakeMethodSimple() {
       return fakeMethodSimple;
     }
 
-    public PageStreamingCallSettings<Integer, Integer, FakePagedListResponse>
-        fakeMethodPageStreaming() {
+    public PagedCallSettings<Integer, Integer, FakePagedListResponse> fakeMethodPageStreaming() {
```

Comment:
Yeah, I think it will be confusing to retain the usage of "page streaming". In the cases where we reference "page streaming" as a noun, reword the docs to use it as an adjective instead.